cmd/shield/commands/info/usage.go

```diff
@@ -71,6 +71,7 @@ func printEnvVars() {
 	header("ENVIRONMENT VARIABLES")
 	contents("SHIELD_TRACE\t\tset to 'true' for trace output.")
 	contents("SHIELD_DEBUG\t\tset to 'true' for debug output.")
+	contents("SHIELD_API_TOKEN\t\tset to send API token on requests")
```

Comment:
May want to mention that it should be set to the Token Value itself.  It kind of sounds like its a boolean (i.e. "if set, send the API token!")